trailblazer/services/analysis_service/analysis_service.py

```diff
@@ -74,6 +74,10 @@ def update_ongoing_analyses(self) -> None:
         analyses: list[Analysis] = self.store.get_ongoing_analyses()
         for analysis in analyses:
             self.job_service.update_jobs(analysis.id)
+
+            if analysis.workflow == WorkflowManager.TOWER:
+                continue
```

Comment:
Could probably move this. Given that the jobs are updated correctly by the job service, I think it might be better to just patch the analysis status update based on the workflow manager